src/js/extensions/ccf/crypto.cpp

```diff
@@ -1010,12 +1010,12 @@ namespace ccf::js::extensions
             ctx, "Unsupported hash algorithm, supported: SHA-256");
         }
 
-        if (algo_name != "RSASSA-PKCS1-v1_5" && algo_name != "ECDSA")
+        if (algo_name != "RSASSA-PSS" && algo_name != "ECDSA")
```

Comment:
That doesn't actually seem to allow EdDSA, am I missing something?

Reply:
I think it's handled above

```
      // Handle algorithms that don't use algo_hash here
      if (*algo_name_str == "EdDSA")
      {
        try
        {
          return JS_NewBool(
            ctx,
            verify_eddsa_signature(
              data, data_size, signature, signature_size, *key_str));
        }
        catch (const std::exception& ex)
        {
          return JS_ThrowRangeError(
            ctx, "Failed to verify EdDSA signature: %s", ex.what());
        }
      }

```